dz02/dz02/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace dz02
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            {
+                Console.WriteLine("Введите через Enter значения x, y центра квадрата");
+                double xCenter = Convert.ToInt32(Console.ReadLine());
+                double yCenter = Convert.ToInt32(Console.ReadLine());
```

Comment:
Вы преобразуете вводимые координаты в целые числа. Это неправильно.
```suggestion
                double xCenter = double.Parse(Console.ReadLine());
                double yCenter = double.Parse(Console.ReadLine());
```